back/sssscs/src/main/java/com/ib/certificate/Certificate.java

```diff
@@ -13,7 +13,6 @@
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import jakarta.persistence.ManyToOne;
```

Comment:
This 'certificate' package has 9 classes. Move some classes to a subpackage or something

Reply:
Fixed in [b111018](https://github.com/magley/sssscs/pull/5/commits/b111018d31a0b6a52e6ee80782ef789c34c47a82).